sdk/tables/data-tables/src/TableTransaction.ts

```diff
@@ -285,7 +286,7 @@ export class InternalTableTransaction {
 
     return tracingClient.withSpan(
       "TableTransaction.submitTransaction",
-      {} as OperationOptions,
+      options,
       async (updatedOptions) => {
         const request = createPipelineRequest({
```

Comment:
Seems like the only property of `updatedOptions` being used in the span is `tracingOptions`. Might need a bit more plumbing to get the other properties on OperationOptions to do anything?

Reply:
Bless you Timo for actually having a brain after the holidays (unlike myself.) Pushed a fix 👍